crates/matrix-sdk/src/event_handler/mod.rs

```diff
@@ -142,15 +151,20 @@ impl HandlerKind {
 }
 
 /// A statically-known event kind/type that can be retrieved from an event sync.
-pub trait SyncEvent {
-    #[doc(hidden)]
+#[doc(hidden)]
+pub trait SyncEvent: Sized {
     const KIND: HandlerKind;
     #[doc(hidden)]
     const TYPE: Option<&'static str>;
+
+    #[doc(hidden)]
+    type Raw: ?Sized;
+    #[doc(hidden)]
```

Comment:
nit: the `#[doc(hidden)]` might not be necessary anymore b/o the one that's on the trait item

Reply:
Oh, `#[doc(hidden)]` on the trait was a mistake. It's still important for users to be able to look up implementers in the docs.